hack/make/test-unit

```diff
@@ -25,15 +25,30 @@ bundle_test_unit() {
 	else
 		TEST_PATH=./${TESTDIRS}
 	fi
-	pkg_list=$(go list -e \
-		-f '{{if ne .Name "github.com/docker/docker"}}
-			{{.ImportPath}}
-		    {{end}}' \
-		"${BUILDFLAGS[@]}" $TEST_PATH \
-		| grep github.com/docker/docker \
-		| grep -v github.com/docker/docker/vendor \
-		| grep -v github.com/docker/docker/man \
-		| grep -v github.com/docker/docker/integration-cli)
+
+	if [ "$(go env GOHOSTOS)" = 'solaris' ]; then
+		pkg_list=$(go list -e \
+			-f '{{if ne .Name "github.com/docker/docker"}}
+				{{.ImportPath}}
+			    {{end}}' \
+			"${BUILDFLAGS[@]}" $TEST_PATH \
+			| grep github.com/docker/docker \
+			| grep -v github.com/docker/docker/vendor \
+			| grep -v github.com/docker/docker/daemon/graphdriver \
```

Comment:
@LK4D4 : sorry misunderstood your question the first time. The issue is with graphtest/graphtest_unix.go. Enabling that for Solaris calls into devmapper for some reason and that fails and then terminates the entire test run. Please remove the Solaris build tag from graphtest_unix.go for now. I will investigate and fix that in a follow up. 
Once the solaris build tag is removed frm that file hack/make/test-unit can go back to the way it was.